.circleci/config.yml

```diff
@@ -1091,13 +1091,13 @@ workflows:
       #          requires:
       #            - wait-calypso-live
       #          test-flags: '-C -S $CIRCLE_SHA1'
-      - test-e2e-canary:
-          name: test-e2e-canary-ie
-          requires:
-            - wait-calypso-live
-          test-flags: '-z -S $CIRCLE_SHA1'
-          test-target: 'IE11'
-          slack-webhook: '$SLACK_IE'
+      # - test-e2e-canary:
+      #     name: test-e2e-canary-ie
+      #     requires:
+      #       - wait-calypso-live
+      #     test-flags: '-z -S $CIRCLE_SHA1'
+      #     test-target: 'IE11'
+      #     slack-webhook: '$SLACK_IE'
       - test-e2e-canary:
```

Comment:
While we are in this area we could as well delete this. It is one single test testing the signup flow in Safari, hardly something we want to do on _every commit_.

I think that will make `wait-calypso-live` redundant, which takes ~10 minutes in CircleCI, freeing up resources.

Reply:
Sure, will do.